qt/python/mantidqt/widgets/codeeditor/multifileinterpreter.py

```diff
@@ -174,3 +184,15 @@ def plus_button_clicked(self, _):
     def save_current_file(self):
         """Save the current file"""
         self.current_editor().save()
+
+    def open_files_in_new_tabs(self, filepaths):
+        for filepath in filepaths:
+            self.open_file_in_new_tab(filepath)
+
+    def restore_session_tabs(self):
```

Comment:
`prev_session_tabs` seems to only be used for restoring? If `MultiFileEditor.readSettings` reads the tabs out could it not pass it directly to `open_files_in_new_tabs` without the need for `restore_session_tabs` at this level? The saving currently only operates at the workbench level so it would be good to keep the session concept out of the widget itself if possible. If required I would keep `prev_session_tabs` on `MultiFileEditor` plugin.